packages/twenty-front/src/modules/command-menu/components/CommandMenuTopBar.tsx

```diff
@@ -69,19 +79,25 @@ export const CommandMenuTopBar = () => {
     contextStoreCurrentObjectMetadataIdComponentState,
   );
 
+  const commandMenuPage = useRecoilValue(commandMenuPageState);
+
   return (
     <StyledInputContainer>
```

Comment:
The container should be non-shrinkable.


https://github.com/user-attachments/assets/cbbaadcf-8fe1-48fc-99ed-4594564beb59

Fix:




https://github.com/user-attachments/assets/efde9cec-5d75-4738-a77c-c8c07ff69e18